bazel/proxy_wasm_cpp_host.patch

```diff
@@ -364,3 +78,17 @@ index aaf7bd2..b1234567 100644
 
 -- 
 2.50.0.727.gbf7dc18ff4-goog
+
+diff --git a/include/proxy-wasm/wasm_vm.h b/include/proxy-wasm/wasm_vm.h
+index 9a2f0a6..a954f0c 100644
+--- a/include/proxy-wasm/wasm_vm.h
++++ b/include/proxy-wasm/wasm_vm.h
+@@ -161,7 +161,7 @@ struct WasmVmIntegration {
+   virtual proxy_wasm::LogLevel getLogLevel() = 0;
+   virtual void error(std::string_view message) = 0;
+   // Allow integrations to handle specific FailStates differently.
+-  virtual void error(FailState fail_state, std::string_view message) { error(message); }
```

Comment:
Simpler thing is to just erase the variable name: `error(FailState, std::string_view)`